<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>614</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="CodeEditor">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>100</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(24, 24, 24);
border-color: rgb(0, 0, 255);
selection-color: rgb(85, 0, 255);</string>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="BottomBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.479, x2:0.948045, y2:0.508, stop:0 rgba(27, 29, 83, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(217, 217, 217);</string>
   </property>
   <widget class="QMenu" name="menuProjects">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuopen">
     <property name="title">
      <string>open</string>
     </property>
     <addaction name="actionfile_2"/>
     <addaction name="actionFolder"/>
    </widget>
    <addaction name="actionsave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actiondiscard"/>
    <addaction name="menuopen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionquit"/>
    <addaction name="actionquit_without_saving"/>
   </widget>
   <widget class="QMenu" name="menuproject">
    <property name="title">
     <string>project</string>
    </property>
    <addaction name="actionsave_all"/>
    <addaction name="actionProjectManager"/>
   </widget>
   <widget class="QMenu" name="menuprefrances">
    <property name="title">
     <string>prefrances</string>
    </property>
    <addaction name="actionsyntax_highlighting"/>
    <addaction name="actionfont_and_stuff"/>
    <addaction name="actionStyle"/>
    <addaction name="actionother"/>
   </widget>
   <addaction name="menuProjects"/>
   <addaction name="menuEdit"/>
   <addaction name="menuproject"/>
   <addaction name="menuprefrances"/>
  </widget>
  <widget class="QToolBar" name="LeftBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 38, 38);
color: rgb(193, 193, 193);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actiondiscard">
   <property name="text">
    <string>discard</string>
   </property>
  </action>
  <action name="actionproject">
   <property name="text">
    <string>project</string>
   </property>
  </action>
  <action name="actionfile">
   <property name="text">
    <string>file</string>
   </property>
  </action>
  <action name="actionproject_2">
   <property name="text">
    <string>project</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionsave_Project_files">
   <property name="text">
    <string>save Project files</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionquit_without_saving">
   <property name="text">
    <string>quit without saving</string>
   </property>
  </action>
  <action name="actionsave_all">
   <property name="text">
    <string>save all</string>
   </property>
  </action>
  <action name="actionProjectManager">
   <property name="text">
    <string>Project Manager</string>
   </property>
  </action>
  <action name="actionsyntax_highlighting">
   <property name="text">
    <string>syntax highlighting</string>
   </property>
  </action>
  <action name="actionfont_and_stuff">
   <property name="text">
    <string>font and stuff</string>
   </property>
  </action>
  <action name="actionStyle">
   <property name="text">
    <string>Style</string>
   </property>
  </action>
  <action name="actionother">
   <property name="text">
    <string>other</string>
   </property>
  </action>
  <action name="actionfile_2">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
